app/assets/stylesheets/notes.scss

```diff
@@ -0,0 +1,19 @@
+.name-date-time {
+	padding: 8px;
+  line-height: 1.42857143;
+  vertical-align: top;
+  border-top: 1px solid #ddd;
+  font-size: 14px;
+  color: #323a45;
+  text-transform: uppercase;
+  letter-spacing: 1.5px;
+  background: #f1f1f1;
+  display: inline-block;
+  float: left;
+  width: 100%;
+  font-weight: bold;
+}
+
+.notes-body {
```

Comment:
Not sure this is doing anything (for me, on chrome, anyway) -- what's the intent here?